network/bridge.go

```diff
@@ -46,22 +50,23 @@ var _ Bridger = (*etcNetworkInterfacesBridger)(nil)
 // 16.04 xenial:   python 3 only (3.5.1)
 //
 // going forward:  python 3 only
-func bestPythonVersion() string {
+func pythonInterpreters() []string {
+	result := []string{}
 	for _, version := range []string{
 		"/usr/bin/python2",
 		"/usr/bin/python3",
 		"/usr/bin/python",
 	} {
 		if _, err := os.Stat(version); err == nil {
-			return version
+			result = append(result, version)
```

Comment:
The only small bug is that likely both 'python2' and 'python' exist on the path because that was the official symlink for python2. Its a little weird to call the same one twice.